_rules/SC1-3-5-autocomplete-valid.md

```diff
@@ -0,0 +1,177 @@
+---
+name: Autocomplete valid
+description: |
+  This rule checks that the HTML autocomplete attribute has a correct value
+
+success_criterion:
+- 1.3.5 # Identify Input Purpose
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Wilco Fiers
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any HTML `input`, `select` and `textarea` element with a [non-empty](#non-empty) `autocomplete` attribute, except if one of the following is true:
+
+- The element is not [visible on the page](#visible-on-the-page) and not [exposed to assistive technologies](#exposed-to-assistive-technologies)
+- The element is an `input` element with a `type` property of `hidden`, `button`, `submit` or `reset`
+- The element has a `disabled` or `aria-disabled="true"` attribute
+- The element has `tabindex="-1"` and has a [semantic role](#semantic-role) that is not a [widget](https://www.w3.org/TR/wai-aria-1.1/#widget_roles)
+
+### Expectation 1
+
+The `autocomplete` attribute is a single term, or a space seperated list of terms.
+
+### Expectation 2
+
+The autocomplete term(s) follow the [HTML 5.2 specification](https://www.w3.org/TR/html52/sec-forms.html#autofill-detail-tokens), which requires that it/they match the following:
+
+1. Has a section-* value _(optional)_
+2. Has either "shipping" or "billing" _(optional)_
+3. Has "home", "work", "mobile", "fax" or "pager" _(optional, only for "email", "impp", "tel" or "tel-*")_
```

Comment:
It's not clear what is optional, and what the second set of tokens applies to. 
I think it would be easier to present this as two steps, where the first one is optional and the second one is required. 
I would also prefer to have a full list of tel- starting options instead of tel-* (which might be perceived as a single option with that value)

Reply:
I've made some changes. I don't want to spell out the exact algorithm here. We're linking to it if someone wants the precise description. This is just to help you along.